model/clusters_mgmt/v1/provision_shard_type.model

```diff
@@ -37,4 +37,7 @@ class ProvisionShard {
 
 	// Contains the cloud provider name
 	link CloudProvider CloudProvider
+
+	// Contains the name of Managment Cluster for Hypershift clusters
```

Comment:
Would be nice if you can explain in the documentation that is the currently selected management cluster for this shard, that it will change over time, and that t is read-only, i.e. it can't be changed via the API.

Reply:
Done.